src/vuestic-theme/vuestic-components/va-modal/VaModal.vue

```diff
@@ -131,14 +136,21 @@ export default {
     hasActionsSlot () {
       return this.$slots.actions
     },
+    scrollWidth () {
```

Comment:
This looks over-complicated.
Correct positioning for close button could probably be achieved with flex and paddings alone.